__tests__/app.test.js

```diff
@@ -102,6 +102,7 @@ describe('GET', () => {
                     expect(article.body).toBe("I find this existence challenging");
                     expect(article.created_at).toBe("2020-07-09T20:11:00.000Z");
                     expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+                    expect(article.comment_count).toBe(11);
```

Comment:
It wold be beneficial to have a new test for this rather than refactoring a previous one